### PR TITLE
Cleanup for statistics

### PR DESCRIPTION
**Pull request by guljain**
Cleaning up dead code

### CODE REVIEW

File: gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStatistic.java [hunk 1]
```diff
@@ -39,40 +39,22 @@
...
-      "action_http_post_request_failures", "POST request failure.", TYPE_COUNTER),
-  ACTION_HTTP_DELETE_REQUEST_FAILURE(
-      "action_http_delete_request_failures", "DELETE request failure.", TYPE_COUNTER),
 
   /** FileSystem-level statistics */
   DIRECTORIES_CREATED(
       "directories_created",
       "Total number of directories created through the object store.",
       TYPE_COUNTER),
-  DIRECTORIES_DELETED(
```
Comment (arunkumarchacko) [inline gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStatistic.java:72]: why removing this?

Comment (guljain) [reply gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStatistic.java:72]: Directories Deleted isn't used in master, only functions were defined but never actually incremented.

**Review by arunkumarchacko — APPROVED** [review]

**Pull request merged**